data/js/npc/ai/vendor_bdo_dispenser.js

```diff
@@ -668,6 +668,11 @@ function onDropItemOnNpc( pDropper, npcDroppedOn, iDropped )
 	if( socket == null )
 		return false;
 
+	if( iDropped.id == 0x0eed )
```

Comment:
Change this from **1** to **true** to stay consistent with the "false" elsewhere in the function.

Also, add a comment saying why the script responds to dropping gold on the NPC, such as
`// Return true allows code/other scripts to handle "gold drop" on this NPC`